lib/libutils/ext/include/util.h

```diff
@@ -75,6 +75,8 @@
 /* Round down the even multiple of size, size has to be a multiple of 2 */
 #define ROUNDDOWN(v, size) ((v) & ~((__typeof__(v))(size) - 1))
 
+#define DIV_ROUND_UP(x, y) (((x) + (y) - 1) / (y))
```

Comment:
What's wrong with `ROUNDUP_DIV()`?

Reply:
In the comments, in is indicated that ROUNDUP_DIV y parameter must be a value of 2. In my case it isn't since it's a rate to be set on the clock. If used with numbers which are non multiple of 2, it yields weird results which are non usable.

Reply:
we should add a assertion on `y` value for `ROUNDUP_DIV()`.

Reply:
OK, please give a proper description of the macro so one can understand when to use it and when to use `ROUNDUP_DIV()` instead.